### DISCUSSION TITLE
NameEmail from string with point in name

### DISCUSSION THREAD

**Discussion by derOtto** [pydantic/pydantic#2960] [Question] [answered]
Hey,
don't know if I'm missing something, or if it is a bug or a feature:
I am reading email addresses from a CSV in NameEmail pydantic attribute of a Basemodel class.
Then storing the string in an SQLalchemy field (as a string).
When I later load in the Basemodel class, the validation of the name fails (the sting is something like "foo.bar <foo.bar@example.com>" and in the CSV "foo.bar@example.com").
Without the point in the name it is working, otherwise I get a validation error.

Could I force enabling points in names or to separate the foo.bar in foo bar at creation?
Thanks in advance.

**Comment by Kludex** [comment, marked as answer]
This is working on V2: https://github.com/pydantic/pydantic/issues/2955. 🙏
